portablemc/cli/lang.py

```diff
@@ -81,12 +82,14 @@ def get(key: str, **kwargs) -> str:
     "args.start.version": "Version identifier (default to release): {formats}.",
     "args.start.version.standard": "release|snapshot|<vanilla-version>",
     "args.start.version.fabric": "fabric:[<vanilla-version>[:<loader-version>]]",
+    "args.start.version.legacyfabric": "legacyfabric:[<vanilla-version>[:<loader-version>]]",
     "args.start.version.quilt": "quilt:[<vanilla-version>[:<loader-version>]]",
     "args.start.version.forge": "forge:[<forge-version>] (forge-version >= 1.5.2)",
     "args.start.version.neoforge": "neoforge:[<neoforge-version>] (neoforge-version >= 1.20.1)",
     "args.start.version.comp.release": "Start the latest release (default).",
     "args.start.version.comp.snapshot": "Start the latest snapshot.",
     "args.start.version.comp.fabric": "Start Fabric mod loader with latest release.",
+    "args.start.version.comp.legacyfabric": "Start (legacy) Fabric mod loader with latest release.",
```

Comment:
I'm not sure what to say here, because legacy fabric uses normal fabric loader. There is no LegacyFabric mod loader (anymore, at least)